common/reviews/api/rush-lib.api.ts

```diff
@@ -496,19 +517,30 @@ class VersionMismatchFinder {
 class VersionPolicy {
   constructor(versionPolicyJson: IVersionPolicyJson);
   // (undocumented)
+  public abstract bump(bumpType?: BumpType, identifier?: string): void;
+  // (undocumented)
   public readonly definitionName: VersionPolicyDefinitionName;
   // (undocumented)
   public abstract ensure(project: IPackageJson): IPackageJson | undefined;
   // (undocumented)
+  public readonly json: IVersionPolicyJson;
+  // (undocumented)
   public static load(versionPolicyJson: IVersionPolicyJson): VersionPolicy;
   // (undocumented)
   public readonly policyName: string;
+  // (undocumented)
+  public abstract validate(versionString: string, packageName: string): void;
 }
 
 // @alpha (undocumented)
 class VersionPolicyConfiguration {
   public constructor(private _jsonFileName: string);
   // @alpha
+  public bump(versionPolicyName?: string,
+      bumpType?: BumpType,
+      identifier?: string,
+      shouldCommit?: boolean): void;
```

Comment:
Four parameters are not ideal. But I will leave it for alpha apis. 

---
In reply to: [134069635](https://github.com/Microsoft/web-build-tools/pull/277#discussion_r134069635) [](ancestors = 134069635)